website/static/js/fileViewTreebeard.js

```diff
@@ -63,11 +63,13 @@ function FileViewTreebeard(data) {
             this.options.showTotal = Math.floor(containerHeight / this.options.rowHeight) + 1;
             this.redraw();
         },
-        lazyLoadOnLoad: function(tree) {
+        lazyLoadOnLoad: function(tree, event) {
             var tb = this;
             Fangorn.DefaultOptions.lazyLoadOnLoad.call(tb, tree);
             Fangorn.Utils.findCurrentFileID.call(tb, tree, window.contextVars.node.id, window.contextVars.file);
-            Fangorn.Utils.scrollToFile.call(tb, tb.currentFileID);
+            if(!event) { 
```

Comment:
Yes that makes sense since people reading this code won't know the deal
with event argument.
On Tue, Apr 14, 2015 at 11:29 AM Steven Loria <notifications@github.com>
wrote:

> In website/static/js/fileViewTreebeard.js
> <https://github.com/samanehsan/osf.io/pull/6#discussion_r28337841>:
>
> >              var tb = this;
> >              Fangorn.DefaultOptions.lazyLoadOnLoad.call(tb, tree);
> >              Fangorn.Utils.findCurrentFileID.call(tb, tree, window.contextVars.node.id, window.contextVars.file);
> > -            Fangorn.Utils.scrollToFile.call(tb, tb.currentFileID);
> > +            if(!event) {
>
> @caneruguz <https://github.com/caneruguz> To clarify, event is only a
> non-null value if the user clicks on a folder?
>
> It would be good to add a comment here, as this fix is not obvious.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/samanehsan/osf.io/pull/6/files#r28337841>.
>